.github/workflows/container-image.yml

```diff
@@ -9,69 +11,81 @@ jobs:
+          - targetarch: x86_64
+            arch: amd64
+    permissions:
+      packages: write
+      id-token: write
     needs:
-      - ci
+     - ci
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Setup rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-      - run: rustup target add x86_64-unknown-linux-musl
-      - name: Setup musl for x86_64
+      - name: Configure Ubuntu repositories
         run: |
-          curl https://musl.cc/x86_64-linux-musl-cross.tgz | tar -xz
-          echo "$PWD/x86_64-linux-musl-cross/bin" >> $GITHUB_PATH
-      - name: Build policy-server
-        env:
-          CC: x86_64-linux-musl-gcc
-        run: cargo build --target=x86_64-unknown-linux-musl --release
-      - run: mv target/x86_64-unknown-linux-musl/release/policy-server policy-server-x86_64
-      - name: Upload policy-server
-        uses: actions/upload-artifact@v2
-        with:
-          name: policy-server-x86_64
-          path: policy-server-x86_64
-  build-aarch64:
-    name: Build aarch64 binary
-    runs-on: ubuntu-latest
-    needs:
-      - ci
-    steps:
+          sudo dpkg --add-architecture arm64
+
+          sudo sed -i "s/deb h/deb [arch=amd64] h/g" /etc/apt/sources.list
+
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy main restricted" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-updates main restricted" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy universe" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-updates universe" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy multiverse" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-updates multiverse" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-backports main restricted universe multiverse" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-security main restricted" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-security universe" /etc/apt/sources.list
+          sudo sed -i "$ a deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports jammy-security multiverse" /etc/apt/sources.list
+
+          sudo apt update -y
+      - name: Install Musl and configure gcc spec
+        run: |
+          sudo apt install -y musl-dev:${{ matrix.arch }}
+          sudo sed -i 1d /usr/lib/${{ matrix.targetarch }}-linux-musl/musl-gcc.specs
+          sudo sed -i "s/*cpp_options/+cpp_options/g" /usr/lib/${{ matrix.targetarch }}-linux-musl/musl-gcc.specs
```

Comment:
I would like to add a comment before the 2 lines patching the spec. Something like:
`# patching the .spec file, as by default it has a bug where it tries to set old_cpp_options but it already exists. using *+cpp_options achieves the same desired functionality of appending preexisting options`

Reply:
fixed in https://github.com/kubewarden/policy-server/pull/308/commits/729c17266e0c7180f26724142359f5daed27b6ae